src/sentry/api/serializers/rest_framework/json.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+import six
+from django.utils.translation import ugettext_lazy as _
+from rest_framework.serializers import WritableField, ValidationError
+from sentry.api.serializers.rest_framework.utils_json import json
+
+# JSONField taken from Django rest framework version 3.9.0
+# See https://github.com/encode/django-rest-framework/blob/master/rest_framework/fields.py
+# or https://www.django-rest-framework.org/api-guide/fields/#jsonfield
+# for more information
+
+
+class JSONField(WritableField):
+    default_error_messages = {
+        'invalid': _('Value must be valid JSON.')
+    }
+
+    def __init__(self, *args, **kwargs):
+        self.binary = kwargs.pop('binary', False)
+        super(JSONField, self).__init__(*args, **kwargs)
+
+    def to_native(self, value):
```

Comment:
Also I don't think we will ever use this. Is it worth keeping?

Reply:
If we aren't using it we shouldn't keep it.